examples/client/Locomotion/src/pages/ActiveRide/RideDrawer/RideOptions/ServiceOptions/ServiceCard/index.js

```diff
@@ -113,7 +113,7 @@ const ServiceCard = ({ service, withBorder }) => {
                 : <View />
             }
               <Price>
-                {service.price !== undefined ? serviceDisplayPrice : unavailableText}
+                {!unavailable ? serviceDisplayPrice : unavailableText}
```

Comment:
more readable without the !not i think {unavailable ? unavailableText : serviceDisplayPrice}